### ISSUE TITLE
Absurd behavior with product images

### ISSUE THREAD

**Issue by seriewe** [smartstoreag/SmartStoreNET#1590]
we notice a very strange behavior in our product images. We upload a product image which is displayed in the preview as normal but when zooming in we see trucks or tools like wrenches. The behavior is sporadic, sometimes the problem occures with the Size parameters ?size=600 as soon as it is changed the image is ok, somethimes is the url withouth size param wrong but with size param added we see the right image. 

Here it is e.g:   
URL Shows wrong image here should be our product:
https://sambol.de/media/image/1071/ultraschall-reinigungsgerat-general-sonic-serie-modell-gs6h.jpg 
With parameters we then get to see our product: https://sambol.de/media/image/1071/ultraschall-reinigungsgerat-general-sonic-serie-modell-gs6h.jpg?size=600 

Can someone explain this to me?

**Comment by muratcakir** [comment]
Is the "TinyImage" plugin installed and active?

**Comment by seriewe** [comment]
Yes, the TinyImage plugin is installed and active. The Support Team did some quick-fix for the plugin and deployed to our server, now we are observing the behavior.

**Comment by seriewe** [comment]
Issue is back, but now we get wrong pictures from our store, so we see wrong products for example.
